source/ed/data.h

```diff
@@ -52,10 +53,16 @@ namespace ed {
 template <typename T>
 void normalize_uri(std::vector<T*>& vec) {
     std::string prefix = navitia::type::static_data::get()->captionByType(T::type);
-    for (auto* element : vec) {
-        // Suppression des espaces de l'URI
-        boost::algorithm::replace_all(element->uri, " ", "");
-        element->uri = prefix + ":" + element->uri;
+
+    if (prefix == "stop_point") {
+        LOG4CPLUS_INFO(log4cplus::Logger::getInstance("log"), "Keeping spaces for stop_point: " << vec);
```

Comment:
If this compiles by some magic template deduction, this will log all the stop_points, not just the ones with a space in their uri.
Do you really want to do that ? There can be more than 10 000 stop_points